Makefile

```diff
@@ -1,14 +1,10 @@
 .PHONY: all
 all: test build
 
-internal/agent/agent.pb.go: proto/agent.proto
-	mkdir -p internal/agent
-	protoc -I ./proto agent.proto --go_out=plugins=grpc:internal/agent
+internal/agent/agent.pb.go:
+	go generate ./...
 
-.PHONY: protoc
-protoc: internal/agent/agent.pb.go
-
-epithet-oidc-plugin: cmd/epithet-oidc-plugin/* pkg/oidc/* internal/agent/*
+epithet-oidc-plugin: internal/agent/agent.pb.go cmd/epithet-oidc-plugin/* pkg/oidc/* internal/agent/*
```

Comment:
`internal/agent/*` should cover `internal/agent/agent.pb.go`

Reply:
Why all these individual file dependencies? We are not using makefile rules to decide what gets compiled, we are using go to do that.

Reply:
It's true that we are using `go build`, but I would imagine you usually type `make build` instead of `go build` right? In that case, Make will think there is no change, and just exit.